### PR TITLE
added namespace

### PR DESCRIPTION
**Pull request by krgrsebastian**
# Description
added description to change namespace in lens

### Related issues

#76 

## Type of change

What types of changes does your code introduce to this project?

- [ x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 1]
```diff
@@ -14,12 +14,14 @@ In this chapter, we'll guide you through connecting to our Kubernetes cluster us
...
 2. If you installed the UMH using the management console, you should see a cluster named "k3d-united-manufacturing-hub" under **Browse**. Click on it to connect.
-3. On the right side of UMHLens under **workloads** -> **pods** you can check the status of all pods. Depending on your system, it may take a while for all pods to start.
+3. You can check the status of all pods by navigating to **workloads** -> **pods** and selecting **Namespace: united-manufacturing-hub** as the namespace on the top right. Depending on your system, it may take a while for all pods to start.
 
    ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingPods.png?width=75%)
-4. To access the web interfaces of the microservices, e.g. node-red or grafana, navigate to **network**->**services** on the left-hand side.
+   ![Untitled](/images/getstarted/managingTheSystem/LensSelectNamespace.png)
+4. To access the web interfaces of the microservices, e.g. node-red or grafana, navigate to **network**->**services** on the left-hand side. Again make sure to change the namespace to **Namespace: united-manufacturing-hub** at the top right.
```
Comment (DaruZero) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:21]: use `resource` shortcode for namespace name

**Review by DaruZero — CHANGES_REQUESTED** [review]
Button names should be capitalized. e.g. **Workloads** -> **Pods**

**Comment by Scarjit** [comment]
@JeremyTheocharis Can you check the CLA error here ?

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 2]
```diff
@@ -13,13 +13,18 @@ In this chapter, we'll guide you through connecting to our Kubernetes cluster us
 ##  1. Connect to UMH
 
 1. Download & install UMHLens [here](https://github.com/united-manufacturing-hub/UMHLens/releases).
-2. If you installed the UMH using the management console, you should see a cluster named "k3d-united-manufacturing-hub" under **Browse**. Click on it to connect.
-3. On the right side of UMHLens under **workloads** -> **pods** you can check the status of all pods. Depending on your system, it may take a while for all pods to start.
+2. If you installed the UMH using the management console, you should see a cluster named "k3d-united-manufacturing-hub"
+   under **Browse**. Click on it to connect.
+3. You can check the status of all pods by navigating to **workloads** -> **pods** and selecting
```
Comment (DaruZero) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:18]: Capitalize button names

File: umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md [hunk 3]
```diff
@@ -13,13 +13,18 @@ In this chapter, we'll guide you through connecting to our Kubernetes cluster us
...
+   {{< resource type="ns" name="umh" >}} as the namespace on the top right. Depending on your system, it may take a while for all pods to start.
 
    ![Untitled](/images/getstarted/managingTheSystem/getStartedManagingPods.png?width=75%)
-4. To access the web interfaces of the microservices, e.g. node-red or grafana, navigate to **network**->**services** on the left-hand side.
+   ![Untitled](/images/getstarted/managingTheSystem/LensSelectNamespace.png)
+4. To access the web interfaces of the microservices, e.g. node-red or grafana, navigate to **network**->**services** on
```
Comment (DaruZero) [inline umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md:23]: Capitalize button names

**Review by DaruZero — CHANGES_REQUESTED** [review]
Can you consider opening a new issue regarding the styling of the whole getstarted section?

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**